tests/tpot_tests.py

```diff
@@ -335,16 +348,38 @@ def test_timeout():
     tpot_obj._optimized_pipeline = creator.Individual.from_string(pipeline_string, tpot_obj._pset)
     tpot_obj.fitted_pipeline_ = tpot_obj._toolbox.compile(expr=tpot_obj._optimized_pipeline)
     # test _wrapped_cross_val_score with cv=20 so that it is impossible to finish in 1 second
+    cv = model_selection.KFold(n_splits=20).split(
+        X=training_features_r,
+        y=training_target_r,
+    )
+    cv = model_selection._split.check_cv(cv, training_target_r, classifier=False)
     return_value = _wrapped_cross_val_score(tpot_obj.fitted_pipeline_,
                                             training_features_r,
                                             training_target_r,
-                                            cv=20,
+                                            cv=cv,
```

Comment:
This is necessary because `_wrapped_cross_val_score` will no longer accept an `int` as it's cv parameter. We should consider renaming the `cv` parameter to `cv_splitter` or something to make this distinction more verbose, but that might require a lot of small changes everywhere, I'm not sure if it's worth it.

Reply:
I think keeping `cv` is ok. Please also update docstring around [this line](https://github.com/EpistasisLab/tpot/blob/v0.11.1/tpot/gp_deap.py#L397)

Reply:
Doesn't this change make the docstring _valid_? Before this change, using a cross-validation generator did not work. Now it does. I'm sorry if I am missing something.

Reply:
I think int won't be accepted after this changes. So the docstring need remove `Int` and `If CV is a number, then it is the number of folds to evaluate each` from the `_wrapped_cross_val_score`

Reply:
I'm very sorry. I followed the link and thought I was looking at the documentation for the `TPOT` class instantiation. You are right, the behavior for `_wrapped_cross_val_score` changes. I will update the docstring.

Reply:
I'm thinking of changing the documentation to reflect that `cv` is expected to be an instance of a generator whos items support the `split` method with the signature `split(X, y, groups)`. Something like this:

```python3
cv: cross-validation split generator instance
        Cross-validation split generator instance which generates the train/test
        splits via the ``split`` method with signature ``split(X, y, groups)``
```

Does this make sense?

Reply:
Hmm, we want cv param in TPOT to be consistent with cv param in cross_eval_score from scikit-learn to avoid misunderstanding. So I think maybe this new way is not needed for now although I think it is a good idea.

Reply:
Okay, I will make the docstring simply:
```python3
cv: cross-validation generator
        Object to be used as a cross-validation generator.
```